app/modules/FrontendHttpActionAdaptor.scala

```diff
@@ -0,0 +1,19 @@
+package modules
+
+import org.pac4j.core.context.HttpConstants
+import org.pac4j.play.PlayWebContext
+import org.pac4j.play.http.PlayHttpActionAdapter
+import play.mvc.{Result, Results}
+
+class FrontendHttpActionAdaptor extends PlayHttpActionAdapter {
+
+  override def adapt(code: Int, context: PlayWebContext): Result = {
+    if (code == HttpConstants.UNAUTHORIZED) {
+      Results.unauthorized(views.html.index.render().toString()).as(HttpConstants.HTML_CONTENT_TYPE)
+    } else if (code == HttpConstants.FORBIDDEN) {
+      Results.forbidden(views.html.index.render().toString()).as(HttpConstants.HTML_CONTENT_TYPE)
+    } else {
+      super.adapt(code, context)
+    }
```

Comment:
This will change once we have some proper unauthorised pages but it's ok for a placeholder for now I think.

Reply:
I've [added it to the backlog](https://github.com/orgs/nationalarchives/projects/20#card-31548460) so we don't forget. :+1: